functions/.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
.idea/.gitignore is the branch at the moment, needs to be moved to hide the .idea files